static/src/javascripts/lib/geolocation.js

```diff
@@ -408,6 +408,36 @@ const extendedCurrencySymbol = {
 const getLocalCurrencySymbol = (): string =>
     extendedCurrencySymbol[countryCodeToCountryGroupId(getSync())] || '£';
 
+// A limited set of country names for the test to add country name in the epic copy
+const countryNames = {
```

Comment:
is this based on traffic/acquisition numbers?

Reply:
yes, I think it's something like 98% of contributions